Fix Auto hide Thumb issue #25

racka98 opened the pull request:
This adds `isDraggingList` as a parameter for the `LaunchedEffect` responsible for hiding the scrollbar. It should have no effect on performance, and it fixes issue #25 

Still not sure about the `isHover` logic for showing the scrollbar as I can't get the scrollbar to show when hovering over the list on the emulator.

alexstyl commented:
Thank you for opening this PR. I will have a look asap

> as I can't get the scrollbar to show when hovering over the list on the emulator.

You can run the code in your computer (desktop app) but running `./gradlew demo-scrollarea:run`. There should also be a configuration available in the project already to try out.

alexstyl commented:
@racka98 did you test your code on desktop?

racka98 commented:
@alexstyl just tested on desktop. The same issue was still persistent. I have updated the PR with a solution that works on both desktop and android. Moved the code for showing the the scroll bar and hiding it into a single `LaunchedEffect`.

alexstyl commented:
Thanks for the contribution. It works great :)

Merging this
